### PR TITLE
[Remove Stream usage: source-formatter]Lps 176218

### PR DESCRIPTION
**Pull request by slnn**
https://issues.liferay.com/browse/LPS-176218

### CODE REVIEW

**Comment by slnn** [comment]
ci:test:sf

**Comment by slnn** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
@slnn No regressions. SF issue is unrelated.

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/upgrade/GradleBuildFile.java [hunk 1]
```diff
@@ -114,7 +114,8 @@ public String getSource() {
 	}
 
 	public List<String> getSourceLines() {
-		return Arrays.asList(_source.split(System.lineSeparator()));
+		return ListUtil.copy(
```
Comment (dantewang) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/upgrade/GradleBuildFile.java:117]: @slnn Please try to use ListUtil.fromArray

Comment (dantewang) [reply modules/util/source-formatter/src/main/java/com/liferay/source/formatter/upgrade/GradleBuildFile.java:117]: @slnn Maybe `ListUtil.fromString` can also do the work, but we may need to double check whether it works in Windows.

**Review by dantewang — CHANGES_REQUESTED** [review]
@slnn Please see my comment and resend, thanks!

**Comment by slnn** [comment]
Update and resend in https://github.com/liferay-core-infra/liferay-portal/pull/3648

**Pull request closed without merging**